crates/precompile/src/secp256r1.rs

```diff
@@ -62,15 +65,12 @@ pub fn verify_impl(input: &[u8]) -> Option<()> {
     // x, y: public key
     let pk = &input[96..160];
 
-    // Prepend 0x04 to the public key: uncompressed form
-    let mut uncompressed_pk = [0u8; 65];
-    uncompressed_pk[0] = 0x04;
-    uncompressed_pk[1..].copy_from_slice(pk);
```

Comment:
The 0x04 was because of `from_sec1_bytes` using `untagged_bytes` means we don't need to do this and can avoid this allocation plus the validation of 0x04 tag